src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/SwitchHelper.groovy

```diff
@@ -164,13 +189,13 @@ class SwitchHelper {
         Wrappers.wait(Constants.RULES_INSTALLATION_TIME) {
             def actualHexCookie = []
             for (long cookie : northbound.getSwitchRules(sw.dpId).flowEntries*.cookie) {
-                actualHexCookie.add(Cookie.decode(cookie).toString())
+                actualHexCookie.add(decode(cookie).toString())
             }
             def expectedHexCookie = []
             for (long cookie : sw.defaultCookies) {
-                expectedHexCookie.add(Cookie.decode(cookie).toString())
+                expectedHexCookie.add(decode(cookie).toString())
             }
-            assert actualHexCookie.sort() == expectedHexCookie.sort()
+            assert (actualHexCookie - expectedHexCookie).empty
```

Comment:
what will happen if some rules are missed on the switch? 
For example 
`actualHexCookie` = [1,2]
`expectedHexCookie` = [1,2,3]

will condition `(actualHexCookie - expectedHexCookie).empty` be false positive in this case

Reply:
very true, thx. Will fix this

Reply:
fixed